theme/global/virtuoso-styles.ts

```diff
@@ -0,0 +1,13 @@
+import { css } from 'leather-styles/css';
```

Comment:
I'm not sure if this is a good spot to put these. Maybe I should add them in `ui/shared` ? 

We keep our other global CSS here but these aren't included in `global.ts` and I don't know if it's good to import from `themes`. 

That's why I thought using a class could be a good idea here as it will be included in the global CSS

Reply:
I would say `ui/shared` makes more sense. This is a specific concerns of virtuoso components, moreover the global styles/panda config.

Reply:
Thanks, I'll move it 👍